pkg/base/config.go

```diff
@@ -79,12 +79,16 @@ const (
 	// the jitter fraction. Jittering is done to prevent multiple leases
 	// from being renewed simultaneously if they were all acquired
 	// simultaneously.
-	DefaultTableDescriptorLeaseDuration = 5 * time.Minute
+	DefaultTableDescriptorLeaseDuration = 2 * time.Minute
```

Comment:
No strong reason to reduce this other than the case where a node dies in the middle of a schema change. That's a pretty rare case, and the tradeoff here is that we renew leases more frequently.